logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -68,7 +68,9 @@ data class Conversation(
     val creatorId: String?,
     val receiptMode: ReceiptMode,
     val messageTimer: Duration?,
-    val userMessageTimer: Duration?
+    val userMessageTimer: Duration?,
+    val archived: Boolean,
+    val archivedRef: String
```

Comment:
question: What is `archivedRef`? What is its format? What if it's empty? Wouldn't `null` be clearer than empty?

Reply:
This value is rather strange I must say, from Scala we see it being saved as `Long`, although we receive it as DateTime/Instant String `1970-01-01 ...`.

But not used anywhere apart from setting/getting from db/api 🤷🏻‍♂️ 

Thought on having it as nullable but wasn't so sure, happy to change it tho :)

Reply:
suggestion: make the name reflect the format. 

Something like `archivedDateTime` or similar, for example.

----

suggestion: better nullable than empty.

If the field is not nullable, I'd expect it to be always parseable. I'd expect `DateTime.fromString(archivedDatetime)` to 100% work if it's not null.
Having it fail with `IllegalArgumentException` because it's an empty string could lead to nasty errors.

So by having it nullable, we can easily differentiate between having this value or not. The compiler will help us.

Reply:
Sure! Works out, will do the changes. Thanks!